crates/key-server/src/server.rs

```diff
@@ -194,7 +198,12 @@ impl Server {
             return Err(InternalError::InvalidCertificate);
         }
 
-        let msg = signed_message(pkg_id, &cert.session_vk, cert.creation_time, cert.ttl_min);
+        let msg = signed_message(
```

Comment:
hmm i wonder if we should use a slightly different message to prevent impersonation? e.g., package x registering the mvr name of 0x123 to impersonate package 0x123
(maybe mvr prevents that in some level?)

Reply:
I'm not sure I follow - you mean if an attacker registers a name that's very similar to an existing MVR name?

Reply:
Not necessarily because you can create sub domains, `subname@name/mvr-app`. This can also be written as `subname.name.sui/mvr-app`, but there will always be an `/` in the name end either `@` or `.`, so a package id can never be parsed as an MVR name.